sdk/lib/collection/linked_list.dart

```diff
@@ -4,7 +4,7 @@
 
 part of dart.collection;
 
-/// A specialized double-linked list of elements that extends [LinkedListEntry].
+/// A specialized circular doubly linked list of elements that extends [LinkedListEntry].
```

Comment:
I guess it is circular, in that the first element's previous element is the last.

The `LinkedList` itself doesn't really use that, it just happens to be a property of the links,
so I don't think I want to emphatise it.

If anything, it's more useful saying that it's a double-ended list, which allows operations at both start and end. The link from first to last element is really just an implementation detail of that.